ci/cron/src/Docs.hs

```diff
@@ -163,20 +166,24 @@ update_s3 opts temp vs = do
 
 update_top_level :: DocOptions -> FilePath -> Version -> Maybe Version -> IO ()
 update_top_level opts temp new mayOld = do
-    new_files <- Set.fromList <$> Directory.listFilesRecursive (temp </> show new)
+    new_files <- Set.fromList <$> files_under new
     old_files <- case mayOld of
         Nothing -> pure Set.empty
-        Just old -> Set.fromList <$> Directory.listFilesRecursive (temp </> show old)
+        Just old -> Set.fromList <$> files_under old
     let to_delete = Set.toList $ old_files `Set.difference` new_files
     Control.when (not $ null to_delete) $ do
         putStrLn $ "Deleting top-level files: " <> show to_delete
         Data.Foldable.for_ to_delete (\f -> do
-            proc_ ["aws", "s3", "rm", s3Path opts f, "--recursive"])
```

Comment:
We're explicitly listing only files, not folders, so `--recursive` makes no sense here.